### PR TITLE
[ol]- add pagination and styling

### PR DESCRIPTION
**Pull request by OrlandoLee**
each page now has 25 projects and it's using google style for now.
@sradu PTAL
mobile:
FXw2IBO1i1

desktop:
vggnFOr1xf

### CODE REVIEW

**Comment by sradu** [comment]
Tailwind design is still pending right?

**Comment by OrlandoLee** [comment]
@sradu updated to use Tailwind PTAL

File: app/views/kaminari/_next_page.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<% if !current_page.last? %>
+  <%= link_to raw('<button type="button" class="-ml-px relative inline-flex items-center px-2 py-2 rounded-r-md border border-gray-300 bg-white text-sm leading-5 font-medium text-gray-500 hover:text-gray-400 focus:z-10 focus:outline-none focus:border-blue-300 focus:shadow-outline-blue active:bg-gray-100 active:text-gray-500 transition ease-in-out duration-150">
+          <svg class="h-5 w-5" fill="currentColor" viewBox="0 0 20 20">
```
Comment (sradu) [inline app/views/kaminari/_next_page.html.erb]: Spacing 2 would be nice to keep things unified.

File: app/views/kaminari/_next_page.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<% if !current_page.last? %>
+  <%= link_to raw('<button type="button" class="-ml-px relative inline-flex items-center px-2 py-2 rounded-r-md border border-gray-300 bg-white text-sm leading-5 font-medium text-gray-500 hover:text-gray-400 focus:z-10 focus:outline-none focus:border-blue-300 focus:shadow-outline-blue active:bg-gray-100 active:text-gray-500 transition ease-in-out duration-150">
```
Comment (sradu) [inline app/views/kaminari/_next_page.html.erb]: I would use the approach:

````
<%= link_to url, class: '' do %>
  raw html
<% end %>
````

**Pull request merged**